### PR TITLE
Initail Commit

### PR DESCRIPTION
**Pull request by cs-utkarsh-surana**
#### UTCs:  

- [x] Connector installation verified.  
- [x] Connector logo verified.  
- [x] Actions and Playbook list verified.  
- [x] Playbook tags verified.  
- [x] All Playbook are in info mode verified.  
- [x] All Playbook are in inactive mode verified.

### CODE REVIEW

File: cyolo/constants.py
Comment (cs-prerna-joshi) [inline cyolo/constants.py:1]: remove the empty file.

File: cyolo/info.json [hunk 1]
```diff
@@ -0,0 +1,1371 @@
...
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "title": "Server URL",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "value": "https://console.fortinet-test-1.cyolo.io",
```
Comment (cs-prerna-joshi) [inline cyolo/info.json]: remove the url from value. In placeholder you can give example without company name.

File: cyolo/constants.py
Comment (cs-utkarsh-surana) [reply cyolo/constants.py:1]: Done

File: cyolo/info.json (hunk 1 above)
Comment (cs-utkarsh-surana) [reply cyolo/info.json]: Done

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**